File: CSharpBasics-Exam-26-08-2014-Day/1.TicTacToe-Power.cs
using System;
using System.Numerics;

class TicTacPower
{
    static void Main()
    {
        int x = int.Parse(Console.ReadLine());
        int y = int.Parse(Console.ReadLine());
        int startValue = int.Parse(Console.ReadLine());

        int value = startValue;

        int[,] matrix = new int[3, 3];

        for (int i = 0; i < 3; i++)
        {
            for (int j = 0; j < 3; j++)
            {
                matrix[i, j] = value;
                if(i == y && j == x){
                    Console.WriteLine((BigInteger)(Math.Pow(value, (value + 1) - startValue)));
                }
                value ++;
            }
        }
    }
}

File: JavaScript-SoftUni/Exam-Problems/1.Tic-Tac-Toe-Power.js
(function () {
    function solve(args) {
        'use strict';
        var x = Number(args[0]),
            y = Number(args[1]),
            startValue = Number(args[2]),
            value = startValue,
            matrix = [[],[],[]];

        for (var i = 0; i < 3; i++) {
            for (var j = 0; j < 3; j++) {
                matrix[i][j] = value;
                if(i === y && j === x){
                    console.log(Math.pow(value, ((value + 1)- startValue )));
                }
                value ++;
            }
        }
    }

    var lastTest = [2, 2, 100];
    solve(lastTest);
}());